src/a11y-dialog.ts

```diff
@@ -221,11 +222,12 @@ function $$(selector: string, context: ParentNode = document): HTMLElement[] {
  * Trap the focus inside the given element
  */
 function trapTabKey(el: HTMLElement, event: KeyboardEvent) {
-  const focusableChildren = getFocusableChildren(el)
-  const firstFocusableChild = focusableChildren[0]
-  const lastFocusableChild = focusableChildren.at(-1)
+  const [firstFocusableChild, lastFocusableChild] =
+    getFirstAndLastFocusableChild(el)
```

Comment:
Maybe `getFocusableEdges` for conciseness?

Reply:
On it!